README.rst

```diff
@@ -70,17 +80,10 @@ Known Issues
 * Background tasks are not profiled.
 * Could probably use better transaction naming.
 
-
-Docker
-======
-
-Want to test this module out, but don't have gevent and newrelic installed?  I've got a Docker image for you!
-`hibou/odoo:9.0 <https://hub.docker.com/r/hibou/odoo/>`_
```

Comment:
Maybe keep this and just change the version to 13.0 here

One can always change ODOO_VERSION at https://hub.docker.com/r/hibou/odoo/dockerfile (9.0 was already wrong referenced)

Reply:
Finall call is for @jaredkipe on what he prefers I guess :)

Reply:
Ultimately I'm fine with that, because we run a very different docker setup and build these days.